test/api_test.js

```diff
@@ -263,8 +263,15 @@ suite('API', function() {
     test('checking schema validation', async () => {
       await helper.hooks.createHook('foo', 'bar', hookWithTriggerSchema);
       await helper.hooks.triggerHook('foo', 'bar', {location: 28, 
-        triggeredBy: 'triggerHook'}).then(() => { throw new Error('Location type should be string'); },
-        (err) => { assume(err.statusCode).equals(400); });
+        triggeredBy: 'triggerHook'}).then(() => { throw new Error('Expected an error'); },
+        (err) => { debug('Got expected error: %s', err); });
+    });
+
+    test('checking more than one schema validation error', async () => {
+      await helper.hooks.createHook('foo', 'bar', hookWithTriggerSchema);
+      await helper.hooks.triggerHook('foo', 'bar', {location: 28, otherVariable: 'twelve',
+        triggeredBy: 'triggerHook'}).then(() => { throw new Error('Expected an error'); },
+        (err) => { debug('Got expected error: %s', err); });
```

Comment:
This should probably check that the error was as expected!  Something like `/data.location should be string/.test(err.toString())` and a similar check for the second error.  This context only seems to generate one error -- I think because this patch removes `{format: 'full', verbose: true, allErrors: true}` from the Ajv constructor.

Reply:
yes, you are right! 
all errors -> makes the code return all errors, not only the first
format -> validate formats (data, time, etc)